x/data/server/msg_define_resolver.go

```diff
@@ -0,0 +1,28 @@
+package server
+
+import (
+	"context"
+
+	cosmossdk "github.com/cosmos/cosmos-sdk/types"
+
+	api "github.com/regen-network/regen-ledger/api/regen/data/v1"
+	"github.com/regen-network/regen-ledger/x/data"
+)
+
+// DefineResolver defines a resolver URL and assigns it a new integer ID that can be used in calls to RegisterResolver.
+func (s serverImpl) DefineResolver(ctx context.Context, msg *data.MsgDefineResolver) (*data.MsgDefineResolverResponse, error) {
+	manager, err := cosmossdk.AccAddressFromBech32(msg.Manager)
+	if err != nil {
+		return nil, err
+	}
+
+	id, err := s.stateStore.ResolverInfoTable().InsertReturningID(ctx, &api.ResolverInfo{
+		Url:     msg.ResolverUrl,
+		Manager: manager.Bytes(),
```

Comment:
```suggestion
		Manager: manager,
```
calling `.Bytes()` not necessary

Reply:
committing here and one other instance address in https://github.com/regen-network/regen-ledger/pull/956/commits/5fa93135c4d11c1cde2b44d349c4157587f35923